F/flashrom/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Base.BinaryPlatforms
+
+name = "flashrom"
+version = v"1.2.900"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/flashrom/flashrom.git",
+              "22e9313d908ce376be6686ebeb3f77828ea70c2e")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/flashrom/
+make PREFIX=${prefix} -j${nproc}
+make PREFIX=${prefix} install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = filter(Sys.islinux, supported_platforms())
+
+# ppc64le kernel headers might be broken, disable for now
+platforms = filter(p -> arch(p) != "powerpc64le", supported_platforms())
```

Comment:
Was this meant to be 
```suggestion
platforms = filter(p -> arch(p) != "powerpc64le", platforms)
```
or more simply
```suggestion
filter!(p -> arch(p) != "powerpc64le", platforms)
```
?

Reply:
😳  yes.....